dataproc/nox.py

```diff
@@ -13,37 +13,67 @@
 # limitations under the License.
 
 from __future__ import absolute_import
+
 import os
 
 import nox
 
 
+LOCAL_DEPS = (
+    os.path.join('..', 'api_core'),
+    os.path.join('..', 'core'),
```

Comment:
@chemelnucfin If you check out `dataproc/setup.py`, you'll see only `google-api-core` as a dependency, so `../core` doesn't belong in this list.